src/assets/Nav.css

```diff
@@ -1,4 +1,5 @@
 .nav-bar{
+    font-family: 'Proxima Nova';
```

Comment:
This requires you to install Proxima Nova locally so - https://www.freeproximanova.com/wp-content/uploads/2019/05/Proxima-Nova-Fonts.zip